app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -220,7 +220,10 @@ const CourseCreator = createReactClass({
     const cloneOptions = showNewOrClone ? '' : ' hidden';
     const controlClass = `wizard__panel__controls ${courseFormClass}`;
     const selectClass = showCloneChooser ? '' : ' hidden';
-    const options = this.props.user_courses.map((course, i) => <option key={i} data-id-key={course.id}>{course.title}</option>);
+    const options = this.props.user_courses
+                              .sort((a, b) => b.id - a.id) // Newest to Oldest
```

Comment:
Instead of writing our own sort function, this will be clearer if we use lodash's sortByOrder function: `_.sortByOrder(courses, 'id', 'desc').map(...)`

Reply:
Yes, I agree... that's a better place to do the sorting.